package-lock.json

```diff
@@ -1,14 +1,607 @@
 {
   "name": "clean-css-cli",
   "version": "5.4.2",
-  "lockfileVersion": 1,
```

Comment:
Can you perhaps regenerate this file with older NPM to minimize number of changes? We may update to lockfile v2 later.

Reply:
Alright, I've set the lockfile back to version 1 by using npm v6 in order to install chokidar. Sorry about that, upgrading to v2 was just a side-effect I have not foreseen while I was on npm v7